apps/desktop/desktop/src/main/java/bisq/desktop/main/content/chat/message_container/list/message_box/ChatRulesWarningMessageBox.java

```diff
@@ -56,28 +58,47 @@ public ChatRulesWarningMessageBox(
         learnMoreLink.getStyleClass().addAll("text-fill-green", "font-light", "medium-text");
         learnMoreLink.setOnAction(e -> controller.onLearnMoreAboutChatRules());
 
-        VBox messageBg = new VBox();
-        messageBg.setSpacing(5);
-        messageBg.getChildren().addAll(warningHeadline, message, learnMoreLink);
-        messageBg.setFillWidth(true);
-        messageBg.setAlignment(Pos.CENTER_LEFT);
-        messageBg.getStyleClass().add("system-message-background");
-        HBox.setHgrow(messageBg, Priority.ALWAYS);
+        VBox messageContentVBox = new VBox();
+        messageContentVBox.setSpacing(5);
+        messageContentVBox.getChildren().addAll(warningHeadline, message, learnMoreLink);
+        messageContentVBox.setFillWidth(true);
+        messageContentVBox.setAlignment(Pos.CENTER_LEFT);
+
+        closeIcon = new Hyperlink("X");
```

Comment:
Better use existing close icons. I guess the `images/icons/close-mini-grey.png` should work.

Reply:
updated